requirements.txt

```diff
@@ -0,0 +1,3 @@
+BeautifulSoup==3.2.1
+lxml==3.4.4
+xbundle==0.1.0
```

Comment:
This isn't ready to merge until this gets added to pypi right?  Do we want to wait until that happens, or just set this to the github URL for now, and change it to pypi once that lands?  I'm fine with doing github for now to unblock the merge.